packages/react-combobox/src/components/ComboButton/renderComboButton.tsx

```diff
@@ -8,6 +8,10 @@ import type { ComboButtonState, ComboButtonSlots } from './ComboButton.types';
 export const renderComboButton_unstable = (state: ComboButtonState) => {
   const { slots, slotProps } = getSlots<ComboButtonSlots>(state);
 
-  // TODO Add additional slots in the appropriate place
-  return <slots.root {...slotProps.root} />;
+  return (
+    <slots.root {...slotProps.root}>
+      <slots.content {...slotProps.content} />
+      <slots.expandIcon {...slotProps.expandIcon} />
+    </slots.root>
```

Comment:
Because Combobox has designs for tags like this, it needs the root to not be the element with `role="combobox"`, hence the need for a separate primary content slot:

![screenshot of pill selection in a collapsed combobox button](https://user-images.githubusercontent.com/3819570/161710783-4f8b1dd3-6751-401d-8bff-e964bbd1c854.png)